java/src/org/openqa/selenium/support/events/EventFiringDecorator.java

```diff
@@ -233,12 +233,10 @@ private void fireBeforeEvents(
 
     String methodName = createEventMethodName("before", method.getName());
 
-    int argsLength = args != null ? args.length : 0;
-    Object[] args2 = new Object[argsLength + 1];
+    args = args != null ? args : new Object[0];
+    Object[] args2 = new Object[args.length + 1];
     args2[0] = target.getOriginal();
-    for (int i = 0; i < argsLength; i++) {
-      args2[i + 1] = args[i];
-    }
```

Comment:
Is this more readable? I am not sure how this improves the code, I get the impression that the current version is fine.